app/controllers/concerns/documentation_controller_branding.rb

```diff
@@ -3,6 +3,18 @@
 module DocumentationControllerBranding
   extend ActiveSupport::Concern
 
+  def fetch
+    respond_to do |format|
+      format.json do
+        render json: {
+          helpUrl: 'https://access.redhat.com/products/red-hat-satellite#get-support',
+          upgradeDocsUrl: "#{ForemanThemeSatellite.unversioned_documentation_root}#Upgrade",
```

Comment:
This has 2 relevant links: connected and disconnected. Would it make sense to directly link them to the appropriate pages instead of adding the indirection?

Ideal would be to know if the user is connected or disconnected, but that may be beyond the scope of this PR.

Reply:
Does an external link make sense for disconnected users? If not, maybe we could use direct link here for connected users only instead of indirect page?..

Reply:
Good point about disconnected installs.